cmd/snap-update-ns/change_test.go

```diff
@@ -53,7 +54,38 @@ func (s *changeSuite) SetUpTest(c *C) {
 	// Mock and record system interactions.
 	s.sys = &testutil.SyscallRecorder{}
 	s.BaseTest.AddCleanup(update.MockSystemCalls(s.sys))
+
+	// MountProfileUpdateContext interface compatible update context with a functional implementation
+	// for method Assumptions. Other methods should not be called.
 	s.as = &update.Assumptions{}
+	lock := func() (unlock func(), err error) {
+		c.Errorf("Unexpected call to \"Lock\"")
+		return nil, nil
+	}
+	assumptions := func() *update.Assumptions {
+		return s.as
+	}
+	loadDesiredProfile := func() (*osutil.MountProfile, error) {
+		c.Errorf("Unexpected call to \"LoadDesiredProfile\"")
+		return &osutil.MountProfile{}, nil
+	}
+	loadCurrentProfile := func() (*osutil.MountProfile, error) {
+		c.Errorf("Unexpected call to \"LoadCurrentProfile\"")
+		return &osutil.MountProfile{}, nil
+	}
+	saveCurrentProfile := func(*osutil.MountProfile) error {
+		c.Errorf("Unexpected call to \"SaveCurrentProfile\"")
+		return nil
+	}
+	uid := func() int {
+		c.Errorf("Unexpected call to \"UID\"")
+		return 0
+	}
+	gid := func() int {
+		c.Errorf("Unexpected call to \"GID\"")
```

Comment:
```suggestion
		c.Errorf(`Unexpected call to "GID"`)
```

Reply:
This code was reverted for another reason. Solved